docs/content/reference/install-configuration/providers/kv/etcd.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Traefik Etcd Documentation"
+description: "Use etcd as a provider for configuration discovery in Traefik Proxy. Automate and store your configurations with Etcd. Read the technical documentation."
+---
+
+# Traefik & etcd
+
+## Configuration Example
+
+You can enable the etcd provider with an endpoint as detailed below:
+
+```yaml tab="File (YAML)"
+providers:
+  etcd:
+    endpoints:
+      - "127.0.0.1:2379"
+```
+
+```toml tab="File (TOML)"
+[providers.etcd]
+  endpoints = ["127.0.0.1:2379"]
+```
+
+```bash tab="CLI"
+--providers.etcd.endpoints=127.0.0.1:2379
+```
+
+## Configuration Options 
+
+| Field | Description                                               | Default              | Required |
+|:------|:----------------------------------------------------------|:---------------------|:---------|
+| `providers.providersThrottleDuration` | Minimum amount of time to wait for, after a configuration reload, before taking into account any new configuration refresh event.<br />If multiple events occur within this time, only the most recent one is taken into account, and all others are discarded.<br />**This option cannot be set per provider, but the throttling algorithm applies to each of them independently.** | 2s  | No |
+| `providers.etcd.endpoints` | Defines the endpoint to access etcd. |  `127.0.0.1:2379`     | Yes   |
```

Comment:
@nmengin Shouldn't we change our stance when documenting options that are "required" but have a default value?
My point is it feels contradictory: why is there even a default value if it is required to configure this option.
I suggest that when an option has a meaningful default value it should be documented as not required.

Reply:
I can't speak for @nmengin but here's what i think while `providers.etcd.endpoints` & others does have a default value of 127.0.0.1:XXXX, this default is probably intended for local testing and isn’t suitable for most real-world deployments. Marking this option as “Required” highlights that users need to specify their actual etcd endpoints for the system to function correctly. 

If we label it as “Not Required,” users might assume they don’t need to configure it, which could lead to misconfigurations or connection issues. Therefore, keeping it marked as “Required” ensures users are aware that they need to provide a proper value for their specific environment.

Reply:
Thank you for bringing this up—it’s a great point. I agree that if the default value of an option isn’t meaningful or useful for most users, we should leave it unset and require users to provide their own value.

In the case of `providers.etcd.endpoints`, the default value of `127.0.0.1:2379` might not be appropriate for many deployments since users often need to connect to a remote etcd cluster rather than a local one. To prevent confusion and potential misconfigurations, it makes sense to:

- Set the default value of `providers.etcd.endpoints` to an empty string or nil.
- Clearly mark the option as “Required.”
- Update this for other providers that are similar

For your suggestion on type indication, yeah we are probably not doing a good job with indicating the correct type or format that the option expects. Might be a lot of work at this point but it would make sense to include a new column with title "default value type" to indicate the format the option expect. 

I'll need to do this for every configuration option table that we currently have in this new reference section. 

please let me know what you think?